fusor-ember-cli/app/controllers/review/progress/overview.js

```diff
@@ -2,5 +2,59 @@ import Ember from 'ember';
 import ProgressBarMixin from "../../../mixins/progress-bar-mixin";
 
 export default Ember.Controller.extend(ProgressBarMixin, {
+  needs: ['deployment'],
+
+  isOpenStack: Ember.computed.alias("controllers.deployment.isOpenStack"),
+
+  getOpenstackParamValue: function(paramName, params) {
+    var paramValue = null;
+    var numParams = params.get('length');
+    for (var i=0; i<numParams; i++) {
+      var param = params.objectAt(i);
+      if (param.get('id') === paramName) {
+        paramValue = param.get('value');
+        break;
+      }
+    }
+    return paramValue;
+  },
```

Comment:
@tzumainn, usually in a controller there's a `}.property()` at the end. If it's a function, then I think it should good in the `actions` hash